extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -131,118 +131,118 @@ public function adContextDataProvider() {
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgWikiDirectedAtChildrenByFounder'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgWikiDirectedAtChildrenByFounder' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [
 					'newWikiCategories' => [ 'test' ],
 					'esrbRating' => 'ec'
 				]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgWikiDirectedAtChildrenByStaff'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgWikiDirectedAtChildrenByStaff' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [
 					'newWikiCategories' => [ 'test' ],
 					'esrbRating' => 'ec'
 				]
 			],
 			[
 				'titleMockType' => 'mainpage',
-				'flags' => [],
-				'expectedOpts' => [],
+				'flags' => [ ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ], 'pageType' => 'home' ]
 			],
 			[
 				'titleMockType' => 'search',
-				'flags' => [],
-				'expectedOpts' => ['pageType' => 'search'],
+				'flags' => [ ],
+				'expectedOpts' => [ 'pageType' => 'search' ],
 				'expectedTargeting' => [
 					'newWikiCategories' => [ 'test' ],
 					'pageType' => 'search',
 					'pageName' => 'Special:Search'
 				]
 			],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'tv',
 				'expectedMappedVertical' => 'ent'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'games',
 				'expectedMappedVertical' => 'gaming'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'books',
 				'expectedMappedVertical' => 'ent'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'comics',
 				'expectedMappedVertical' => 'ent'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'lifestyle',
 				'expectedMappedVertical' => 'life'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'not-existing',
 				'expectedMappedVertical' => 'error'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'oldVertical' => 'Wikia',
 				'newVertical' => 'other',
-				'expectedMappedVertical' => 'wikia']
+				'expectedMappedVertical' => 'wikia' ]
 			],
 
-			array_merge($defaultParameters, [
+			array_merge( $defaultParameters, [
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test', 'test1' ] ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "test1" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)